### PR TITLE
finish main part of delete policy

### PR DESCRIPTION
**Pull request by AmosChenYQ**
finish remove policy

### CODE REVIEW

**Comment by AmosChenYQ** [comment]
@hsluoyz

**Comment by hsluoyz** [comment]
@Zxilly plz review.

**Comment by AmosChenYQ** [comment]
plz wait a minute, i will push my new commit. but ECS for developer seems to be in a network problem...

**Review by Zxilly — APPROVED** [review]

File: casbin_pymongo_adapter/__init__.py [hunk 1]
```diff
@@ -1 +1,3 @@
-from .adapter import Adapter
\ No newline at end of file
+from .adapter import Adapter
+# TODO: remove this line
```
Comment (Zxilly) [inline casbin_pymongo_adapter/__init__.py]: plz also remove unused comment

**Review by leeqvip — APPROVED** [review]

**Comment by AmosChenYQ** [comment]
I have removed these 2 lines @Zxilly

**Review by Zxilly — APPROVED** [review]

**Pull request merged**